intelmq/bots/collectors/twitter/collector_twitter.py

```diff
@@ -80,11 +81,12 @@ def init(self):
         self.session = create_request_session(self)
 
     def get_text_from_url(self, url: str) -> str:
-        if "pastebin.com" in url:
+        netloc = urlsplit(url)
+        if netloc == "pastebin.com":
```

Comment:
In theory the netloc could also contain the port, so `netloc.startswith('pastebin.com')` would also work in case the url is `https://pastebin.com:443/...`

Reply:
But it should not match `pastebin.com.example.com`, so we need to split the port off and compare to exact match (or: endswith). And to not split IPv6-Addresses, we need `rsplit(':', maxsplit=1)`